src/main/java/sirius/db/mixing/EntityDescriptor.java

```diff
@@ -215,6 +219,39 @@ protected EntityDescriptor(Class<?> type) {
         return type;
     }
 
+    /**
+     * Builds a standard message that the given entity cannot be saved.
+     * <p>
+     * This message is mostly used as a consequence describing why an error happened.
+     * For instance, when an entity's required field is not populated, a standard message like this will be logged:
+     * {@code Field 'MyField' is required.}
+     * As to would be rather nice to tell the end user that an entity couldn't be saved due to the error.
+     * {@code MyEntity cannot be saved: Field 'MyField' is required.}
+     *
+     * @param clazz the entity class to extract its label
+     * @return the formatted message
+     */
+    public static String createCannotSaveMessage(@Nonnull Class<?> clazz) {
```

Comment:
do we need this? this is just a shortcut for mixing.getDescriptor(clazz).createCannotSaveMessage()

Reply:
I think is useful boilerplate if you have an entity at hand. Otherwise you must (at least) initialize a part...